srcpkgs/SPIRV-Headers/template

```diff
@@ -1,7 +1,7 @@
 # Template file for 'SPIRV-Headers'
 pkgname=SPIRV-Headers
 reverts="1.3.37_1 1.3.44_1 1.3.47_1 1.3.51_1 1.3.60_1 1.3.65_1 1.3.70_1 1.3.74_1"
```

Comment:
I think this line can go as the new version is higher anyway.

Reply:
Makes sense.